jwst/coron/median_replace_img.py

```diff
@@ -1,42 +1,57 @@
-"""Replace bad pixels with the median of the surrounding pixel and median fill
-   the input images.
- """
+"""Replace bad pixels in the input images with the median of the surrounding pixels.
+"""
+
 import logging
 import numpy as np
 
-from jwst.datamodels import dqflags
-
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
-def median_fill_value(input_array, input_dq_array, bsize, xc, yc):
+
+def median_fill_value(input_array, input_dq_array, bsize, bad_bitvalue, xc, yc):
+    """Calculates the median value of good pixels in a cutout of the input array.
+
+    Parameters
+    ----------
+    input_array : ndarray
+        Input array to filter
+
+    input_dq_array : ndarray
+        Input data quality array
+
+    bsize : scalar
+        Box size of the data to extract
+
+    bad_bitvalue : int
+        The sum of all of the DQ bit values to consider bad. Setting to 0
+        will treat all pixels as good.
+
+    xc: scalar
```

Comment:
Really minor nit, but should have a whitespace preceding the colon (i.e. "xc : scalar") to follow conventions. Same in line 32 for `yc`.

Reply:
whoops - fixed!